### PR TITLE
Set up slurm for travis

### PR DESCRIPTION
**Pull request by arjoly**
This issue will solve of https://github.com/clusterlib/clusterlib/issues/3 and also  https://github.com/clusterlib/clusterlib/issues/6.

### CODE REVIEW

**Comment by arjoly** [comment]
@ogrisel The slurm installation is working, however there is a compatibility between python 2 and 3. By chance, do you know a way to perform string comparison that is compatible between python 2 and 3.

File: Makefile [hunk 1]
```diff
@@ -4,6 +4,8 @@ all: clean inplace test
 
 clean:
 	python setup.py clean
+	scontrol show node
```
Comment (arjoly) [inline Makefile]: To remove.

File: .travis.yml [hunk 2]
```diff
@@ -5,9 +5,16 @@ python:
   - "3.4"
 # command to install dependencies
 install:
-  - "python setup.py install"
+  # Install package and its dependency for full test
   - "pip install coverage coveralls"
   - "pip install sphinx_bootstrap_theme"
+  # Install slurm
+  - "sudo apt-get install munge slurm-llnl "
```
Comment (arjoly) [inline .travis.yml]: To check that the munge package is required.

**Comment by ogrisel** [comment]
> @ogrisel The slurm installation is working, however there is a compatibility between python 2 and 3. By chance, do you know a way to perform string comparison that is compatible between python 2 and 3.

The problem is not the comparison: the bytes should be decoded before, right after you read them (e.g. from a socket or a file). In Python 2 is done implicitly with a random (platform specific) encoding. In Python 3 youhave  to decode bytes objects into unicode string explicitly with an appropriate encoding depending on where the bytes come from.

**Comment by arjoly** [comment]
Thanks ! It's working now!

**Comment by arjoly** [comment]
This is merged. Thanks @ogrisel !!!

**Pull request merged**